content/docs/user-guide/gems/reference/aws/aws-gamelift/flexmatch/cpp-api.md

```diff
@@ -0,0 +1,257 @@
+
+To Accept or reject the match, call `AWSGameLiftClientManager::AcceptMatch()` or `AWSGameLiftClientManager::AcceptMatchAsync()`, and pass in a reference to the AcceptMatch request, which contains the player decision. 
+If any player rejects the match, or if acceptances are not received before a specified timeout, the proposed match is dropped.
+
+```cpp
+// For example, make synchronous call to accept the proposed matchmaking
+AWSGameLift::AWSGameLiftAcceptMatchRequest request;
+request.m_acceptMatch = true;
+request.m_ticketId = "YourMatchmakingTicketId";
+request.m_playerIds = { "CurrentPlayerId" };
+
+AWSGameLift::AWSGameLiftMatchmakingRequestBus::BroadcastResult(result, &AWSGameLift::AWSGameLiftMatchmakingRequestBus::Events::AcceptMatch, request);
+
+// For example, make asynchronous call to accept the proposed matchmaking
+AWSGameLift::AWSGameLiftAcceptMatchRequest request;
+request.m_acceptMatch = true;
+request.m_ticketId = "YourMatchmakingTicketId";
+request.m_playerIds = { "CurrentPlayerId" };
+AWSGameLift::AWSGameLiftMatchmakingAsyncRequestBus::Broadcast(&AWSGameLift::AWSGameLiftMatchmakingAsyncRequestBus::Events::AcceptMatchAsync, request);
+
+void OnAcceptMatchAsyncComplete()
+{
+    ...
+}
+```
+
+#### `StartPolling`
+
+Request to start process for polling matchmaking ticket based on given ticket id and player Id. 
+
+To start polling the matchmaking ticket status after matchmaking is started, call `AWSGameLiftClientLocalTicketTracker::StartPolling()` and pass in the matchmaking ticket Id and player Id.
+
+```cpp
+// For example, make synchronous call to start polling
+AZStd::string ticketId = "YourMatchmakingTicketId";
+AZStd::string playerId = "CurrentPlayerId";
+AWSGameLift::AWSGameLiftMatchmakingEventRequestBus::Broadcast(&AWSGameLift::AWSGameLiftMatchmakingEventRequestBus::Events::StartPolling, ticketId, playerId);
+```
+
+#### `StopPolling`
+
+Request to stop process for polling matchmaking ticket.
+
+To stop polling the matchmaking ticket status after matchmaking is completed or canceled, call `AWSGameLiftClientLocalTicketTracker::StopPolling()`.
+
+```cpp
+// For example, make synchronous call to stop polling
+AWSGameLift::AWSGameLiftMatchmakingEventRequestBus::Broadcast(&AWSGameLift::AWSGameLiftMatchmakingEventRequestBus::Events::StopPolling);
+```
+
+{{< caution >}}  
+The local ticket tracker provided by the AWS GameLift Gem is for development only. O3DE developer should replace it with a scalable solution for public release games.
+{{< /caution >}}
+
+
+### Client notifications
+
+Notifications should be sent from the matchmaking ticket tracking system when the ticket status is updated.
+
+#### `OnMatchComplete`
+
+When the matchmaking request is completed, the `AzFramework::MatchmakingNotificationBus::Events::OnMatchComplete()` notification is broadcasted from the matchmaking ticket tracking system. During this step, player can join the game session. 
```

Comment:
nit: Once the matchmaking ticket is completed, the player can join the game session that is hosting the match.